### ISSUE TITLE
Automated minimal quality checks

### ISSUE THREAD

**Issue by FedeClaudi** [brainglobe/bg-atlasgen#13]
Just putting this out here as a possible idea, but perhaps we can add some minimal test for working atlases?
These could include:

1. test on the atlas generation script, formatting, linting and trying to create the atlas (might be hard when it requires data that cannot be downloaded from a website)
2. test on atlas functionality: minimal tests for compatibility with the Atlas API, rendering in Brianrender etc...

Let me know your thoghts!

**Comment by adamltyson** [comment]
Yep, I think this is great idea. Ideally it could run in GH actions too, as part of improving the experience for new contributors.

**Comment by alessandrofelder** [comment]
also this could include a check so that https://github.com/brainglobe/bg-atlasapi/issues/163 can't happen again!
